.github/workflows/test.yml

```diff
@@ -47,7 +47,10 @@ jobs:
       run: export PATH="$PATH:$(go env GOPATH)/bin"; cd _fuzz/it && ./fuzz-ci local-regression
     - name: fuzz continuous job
       run: export PATH="$PATH:$(go env GOPATH)/bin"; cd _fuzz/it && ./fuzz-ci fuzzing
-      if: github.event_name == 'push' && github.ref == 'refs/heads/master'
+      if: >
+        github.event_name == 'push' &&
+        github.ref == 'refs/heads/master' &&
+        github.actor == 'mvdan'
```

Comment:
please use `github.repository == 'mvdan/sh'` instead. otherwise, if I was to give another person push access and they pushed to master, this wouldn't trigger.

Reply:
Done